packages/app/src/__deprecated__/renderer/components/rest/data-modal/data.modals.tsx

```diff
@@ -113,7 +113,7 @@ export const LoadingStateDataModal = ({
   title?: string
   textMessage?: Message
 }) => (
-  <ErrorModal closeButton={false} title={title}>
+  <ErrorModal closeButton={false} title={title} closeable={false}>
```

Comment:
Just wondering - why do we set property on `ErrorModal` while the task description mentions `delete progress modal`?

Reply:
This is deprecated component, `ErrorModal` is the main element for `LoadingStateDataModal` which is used for `delete progress modal`

Reply:
Oh, now I get it